schemas/input/csl-data.json

```diff
@@ -82,7 +82,7 @@
       "title": "EDTF date string",
       "description": "CSL input supports EDTF, levels 0 and 1.",
       "type": "string",
-      "pattern": "^[0-9-%~X?.\/]{4,}$"
+      "pattern": "^[0-9-%~X?./]{4,}$"
```

Comment:
Prettier did this.

Reply:
Ah, I see. It's "which flavor of regex" are you using thing. In JavaScript, "/" doesn't _need_ to be escaped, but in PHP it does.